packages/office-ui-fabric-react/src/utilities/grid/GridCell.Props.ts

```diff
@@ -0,0 +1,74 @@
+export interface IGridCellProps<T> extends React.Props<any> {
+
+  /**
+   * The option that will be made available to the user
+   */
+  item: T;
+
+  /**
+   * Arbitrary unique string associated with this option
+   */
+  id: string;
+
+  /**
+   * Optional, if the this option should be diabled
+   */
+  disabled?: boolean;
+
+  /**
+   * Optional, if the cell is currently selected
+   */
+  selected?: boolean;
+
+  /**
+   * The on click handler
+   */
+  onClick?: (item: T) => void;
+
+  /**
+   * The render callback to handle rendering the item
+   */
+  onRenderItem: (item: T) => JSX.Element;
+
+  /**
+   * Optional, the onHover handler
+   */
+  onHover?: (item?: T) => void;
+
+  /**
+   * Optional, the onFocus handler
+   */
+  onFocus?: (item: T) => void;
+
+  /**
+   * The accessible role for this option
+   */
+  role?: string;
+
+  /**
+   * Optional, className(s) to apply
+   */
+  className?: string;
+
+  /**
+  * Optional, the CSS class used for when the cell is disabled
+  */
+  cellDisabledStyle?: string[];
```

Comment:
Going to postpone doing this since the styling was previously existing and in the old format so the change would be a little out of scope.